src/spatialCapture/Shaders.js

```diff
@@ -314,7 +320,8 @@ export function createPointCloudMaterial(texture, textureDepth, shaderMode, bord
             viewAngleSimilarity: { value: 0.0 },
             viewPositionSimilarity: { value: 0.0 },
 			// used to erase depth in the pre render
-            useFarDepth: {value: false}
+            useFarDepth: {value: false},
```

Comment:
Is useFarDepth used?

Reply:
It's used in vuforia-spatial-toolbox-userinterface/src/spatialCapture/CameraVisPatch.js